### PR TITLE
Button design

### PR DESCRIPTION
**Pull request by aravindraj23**
Add book and add author buttons bootstraped

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -52,4 +59,7 @@
 .active a{
    font-size: 20px;
    color: goldenrod !important;
-}
\ No newline at end of file
+}
+
+
+
```
Comment (MohamedHilaal) [inline app/assets/stylesheets/application.css:65]: Why Empty space

File: app/views/author/add_author.html.erb [hunk 2]
```diff
@@ -16,21 +28,11 @@
     <tr>
       <td><%= count += 1 %></td>
       <td><%= author.name %></td>
-      <td><%= link_to "Edit", edit_author_path(author) %></td>
+      <td><%= link_to 'Edit', edit_author_path(author) %></td>
```
Comment (MohamedHilaal) [inline app/views/author/add_author.html.erb:31]: No need of this changes

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -7,11 +7,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+
+
+
```
Comment (MohamedHilaal) [inline app/views/layouts/application.html.erb:12]: Avoid using space

File: app/views/request/borrow.html.erb [hunk 4]
```diff
@@ -9,11 +9,10 @@
     </tr>
   </thead>
   <tbody>
-      <% count = 0 %>
-      <% @books.each do |book|  %>
+      <% @books.each_with_index do |book, count|  %>
       <% unless book.loaned_books.where(user_id: current_user.id).where("status != 'returned'").exists? %>
         <tr>
-          <td><%= count += 1 %></td>
+          <td><%= count  %></td>
```
Comment (MohamedHilaal) [inline app/views/request/borrow.html.erb:21]: do with index

**Review by MohamedHilaal — APPROVED** [review]

File: app/views/book/edit.html.erb [hunk 5]
```diff
@@ -7,11 +7,11 @@
       </div>  
       <div class= "col-md-3">
         <%= f.select :published_year, 1999.. Date.today.year,
```
Comment (prasanthintrohive) [inline app/views/book/edit.html.erb:9]: add year picker

**Pull request merged**